### PR TITLE
Added a 'get_display' method to FHIR::Definitions

### PR DESCRIPTION
**Pull request by okeefm**
Method takes a code system and code, and returns the 'display' attribute if one is found. If not, returns nil.

### CODE REVIEW

File: lib/fhir_models/bootstrap/definitions.rb [hunk 1]
```diff
@@ -211,6 +211,32 @@ def self.get_codes(uri)
...
 
+    # Get the "display" (human-readable title) for a given code in a code system (uri)
+    # If one can't be found, return nil
+    def self.get_display(uri, code)
+      return nil if uri.nil? || code.nil?
+      load_expansions
+      valuesets = @@expansions.select { |x| !x['compose']['include'].select {|i| i['system'] == uri }.empty? }
+      valuesets += @@valuesets.select { |x| x['url'] == uri }
+      c = nil
+      valuesets.each do |valueset|
```
Comment (wesrich) [inline lib/fhir_models/bootstrap/definitions.rb]: You might consider using .detect instead of .each here, and a couple other places. That would remove the necessity of the breaks I think...

Comment (jawalonoski) [inline lib/fhir_models/bootstrap/definitions.rb]: Good points @wesrich... we should fix that. I guess it'll be another pull request since I've hastily merged it.

Comment (okeefm) [inline lib/fhir_models/bootstrap/definitions.rb]: Yeah, these sound like good updates to me. I'll make them tomorrow and make another pull request. Thanks!
